sdk/iothub/Microsoft.Azure.Management.IotHub/src/Microsoft.Azure.Management.IotHub.csproj

```diff
@@ -1,22 +1,28 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-	<!-- Please do not move/edit code below this line -->
-	<Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
-	<!-- Please do not move/edit code below this line -->
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
 
-	<PropertyGroup>
-		<PackageId>Microsoft.Azure.Management.IotHub</PackageId>
-		<Description>Provides management capabilities for Microsoft Azure IotHub.</Description>
-		<AssemblyTitle>Microsoft Azure IotHub Management</AssemblyTitle>
-		<AssemblyName>Microsoft.Azure.Management.IotHub</AssemblyName>
-		<Version>2.10.0-preview</Version>
-		<PackageTags>Microsoft Azure IotHub;IotHub management;IotHub;</PackageTags>
-		<PackageReleaseNotes>Regenerated IotHub Sdk with updated routing source enum.</PackageReleaseNotes>
-	</PropertyGroup>
-	<PropertyGroup>
-		<TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
-	</PropertyGroup>
-	
-  <!-- Please do not move/edit code below this line -->	
+  <PropertyGroup>
+    <TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
+    <LangVersion>8.0</LangVersion>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.IotHub</PackageId>
+    <Description>Provides management capabilities for Microsoft Azure Iot Hub.</Description>
+    <AssemblyTitle>Microsoft Azure Iot Hub Management</AssemblyTitle>
+    <AssemblyName>Microsoft.Azure.Management.IotHub</AssemblyName>
+    <Version>2.11.0</Version>
+    <PackageTags>Microsoft Azure IotHub;IotHub management;IotHub;</PackageTags>
+    <PackageReleaseNotes>Updated to API version 2020-03-01</PackageReleaseNotes>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
Why do we need the Models folder included separately?

Reply:
I didn't hand include it, but I'll see what happens if I remove it.